### PR TITLE
Updated for release 9.6

### PR DESCRIPTION
**Pull request by vivek-reghunath-sonarsource**
Updated with release 9.6

### CODE REVIEW

File: update-center-source.properties [hunk 1]
```diff
@@ -1,12 +1,20 @@
...
 
 ltsVersion=8.9.9
 
+9.6.description=New rich and context-specific rule descriptions, token expiration, PCI DSS report, encryption for SAML authentication, faster analysis for Java PRs and Kotlin, Kubernetes security rules, JavaScript rules for CDK S3 buckets and React, new rules for AWS in Java and for Azure in .NET
+9.6.downloadUrl=https://binaries.sonarsource.com/Distribution/sonarqube/sonarqube-9.6.0.59041.zip
+9.6.downloadDeveloperUrl=https://binaries.sonarsource.com/CommercialDistribution/sonarqube-developer/sonarqube-developer-9.6.0.59041.zip
+9.6.downloadEnterpriseUrl=https://binaries.sonarsource.com/CommercialDistribution/sonarqube-enterprise/sonarqube-enterprise-9.6.0.59041.zip
+9.6.downloadDatacenterUrl=https://binaries.sonarsource.com/CommercialDistribution/sonarqube-datacenter/sonarqube-datacenter-9.6.0.59041.zip
+9.6.changelogUrl=https://sonarsource.atlassian.net/issues/?jql=project%20%3D%20SONAR%20AND%20fixVersion%20%3D%209.6
```
Comment (wouter-admiraal-sonarsource) [inline update-center-source.properties:15]: You should use the release notes, right?

```suggestion
9.6.changelogUrl=https://sonarsource.atlassian.net/secure/ReleaseNote.jspa?projectId=10139&version=12633
```

**Review by wouter-admiraal-sonarsource — APPROVED** [review]
The release notes link should point to the actual release notes, right? Not the issue search?

File: update-center-source.properties (hunk 1 above)
Comment (vivek-reghunath-sonarsource) [reply update-center-source.properties:15]: As per[ this discussion](https://discuss.sonarsource.com/t/release-note-urls-in-jira-are-broken/11144/4) we earlier decided to use the issues list for release from this release. However, the one you shared is a better solution if we can make it public. Currently, that page appears to be private and needs login. What do you suggest?

Comment (wouter-admiraal-sonarsource) [reply update-center-source.properties:15]: Ah, it thought it was already solved. Well, it's not a big deal, leave it as you wrote 👍

**Review by martin-bednorz-sonarsource — APPROVED** [review]

**Pull request merged**